task_manager/api.py

```diff
@@ -18,23 +18,17 @@
 app = create_app()
 api = Api(app)                          # Create a Flask-RESTPlus API
 
-
-''' HELLO '''
-
-
+@api.route('/')
 @api.route('/hello')                   # Create a URL route to this resource
 class HelloWorld(Resource):            # Create a RESTful resource
+	''' HOME PAGE '''
 	def get(self):                     # Create GET endpoint
 		return {'hello': 'world'}
 
-
-'''
-CREATE NEW TASK
-'''
-
-
+  
```

Comment:
E101: indentation contains mixed spaces and tabs  <!-- Automatically posted by Lintly -->